floodviz/tests/test_hydrograph_utils.py

```diff
@@ -2,41 +2,79 @@
-                                                                                               "siteUsesDaylightSavingsTime": True}, "geoLocation": {"geogLocation": {"srs": "EPSG:4326", "latitude": 42.4077639, "longitude": -92.4632451},
-                                                                                                                                                     "localSiteXY": []}, "note": [], "siteType": [], "siteProperty": [{"value": "ST", "name": "siteTypeCd"}, {"value": "07080205", "name": "hucCd"},
-                                                                                                                                                                                                                      {"value": "19", "name": "stateCd"}, {"value": "19013", "name": "countyCd"}]}, "variable": {"variableCode": [{"value": "00060", "network": "NWIS",
-                                                                                                                                                                                                                                                                                                                                   "vocabulary": "NWIS:UnitValues", "variableID": 45807197, "default": True}], "variableName": "Streamflow, ft&#179;/s", "variableDescription":
-                                                                                                                                                                                                                                                                                                                     "Discharge, cubic feet per second", "valueType": "Derived Value", "unit": {"unitCode": "ft3/s"}, "options": {"option":
-                                                                                                                                                                                                                                                                                                                                                                                                                                      [{"name": "Statistic", "optionCode": "00000"}]}, "note": [], "noDataValue": -999999.0, "variableProperty": [], "oid": "45807197"}, "values":
-                                                           [{"value": [{"value": "446", "qualifiers": ["A"], "dateTime": "2008-05-20T00:00:00.000-05:00"}]}]}}}
-
-        ## Returned value is different than the mocked request response because req_hydrodata accessess two extra levels of the dict before returning.
+        self.mock_response = {
+            "value": {"timeSeries": {"sourceInfo": {"siteName": "Black Hawk Creek at Hudson, IA", "siteCode":
+                [{"value": "05463500", "network": "NWIS", "agencyCode": "USGS"}], "timeZoneInfo": {"defaultTimeZone":
+                {
```

Comment:
Yeah the default formatting is not good. It helps a little if you set the line length to 80 I think. You may have to do this one more by hand. Generally, it's easier to read with one property per line. Don't worry about it now. But if you have time this week, it would be nice to pretty this up.